src/adapter.jl

```diff
@@ -14,8 +14,9 @@ function show_string(io::IO, obj::Union{GapObj,FFE})
       # For objects that do not fit on the screen,
       # show only the first and the last lines.
       upper = div(rows, 2)
-      stri = join(lines[1:upper], "\n") * "\n  ⋮\n" *
-             join(lines[(end-rows+upper+2):end], "\n")
+      stri = join(lines[1:upper], "\n")::String
+      stri *= "\n  ⋮\n"
+      stri *= join(lines[(end-rows+upper+2):end], "\n")::String
```

Comment:
I do not understand why the old code was problematic:
Doesn't the documentation of `join` say that a `String` is returned?

Reply:
I've dropped this for now until I understand it properly.